src/validations/rules/sar.sch

```diff
@@ -176,6 +176,84 @@
 
     </sch:pattern>
 
+    <sch:pattern
+        id="results">
+        <sch:rule
+            context="oscal:observation">
+            <sch:assert
+                diagnostics="has-method-TEST-diagnostic"
+                doc:guide-reference="Guide to OSCAL-based FedRAMP Security Assessment Plans (SAR) §4.7"
```

Comment:
Documentation reference should be "_Guide to OSCAL-based FedRAMP Security Assessment **Results** (SAR)_".